setup.py

```diff
@@ -46,7 +46,7 @@
         'spark': ['pyspark>=2.4.0'],
         'mlflow': ['mlflow>=1.0'],
     },
-    python_requires='>=3.5,<3.9',
+    python_requires='>=3.5,<4.0',
```

Comment:
I would do it as `3.10` instead of `4.0`.

Reply:
Fixed